server/src/main/java/io/crate/planner/GCDanglingArtifactsPlan.java

```diff
@@ -44,7 +41,8 @@ public void executeOrFail(DependencyCarrier dependencies,
                               Row params,
                               SubQueryResults subQueryResults) {
         var listener = OneRowActionListener.oneIfAcknowledged(consumer);
-        DeleteIndexRequest deleteRequest = new DeleteIndexRequest(IndexParts.DANGLING_INDICES_PREFIX_PATTERNS.toArray(new String[0]));
-        dependencies.client().execute(DeleteIndexAction.INSTANCE, deleteRequest).whenComplete(listener);
+        dependencies.transportGCDanglingArtifactsAction()
+            .execute(GCDanglingArtifactsRequest.INSTANCE)
+            .whenComplete(listener);
```

Comment:
Preferred way to make requests would be `dependencies.client().execute(TransportGCDanglingArtifactsAction.ACTION, GCDanglingArtifactsRequest.INSTANCE)` 

This avoids having to add more and more Transport actions to the `DependencyCarrier` and the `NodeClient` already has access to the `actions` map.

Reply:
thanks, https://github.com/crate/crate/pull/17729